guide/14-deep-learning/how_AutoDL_works.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<ul><li>The <code>load</code>&nbsp;method is used to load a saved model from the disk using the&nbsp;<code>AutoDL</code>&nbsp;class. It accepts the following parameters:</li><li class="ql-indent-1">path: Path to the ESRI Model Definition (.emd) file</li><li class="ql-indent-1">data: Returned data object from <code>prepare_data</code> function</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>